src/components/Edit/NatureOfBusiness.vue

```diff
@@ -25,32 +27,49 @@ import { NatureOfBusiness as NatureOfBusinessShared } from '@bcrs-shared-compone
   }
 })
 export default class NatureOfBusiness extends Vue {
-  private NaicsServices = NaicsServices
+  /** Whether to show invalid section styling. */
+  @Prop({ default: false })
+  readonly invalidSection!: boolean
+
+  readonly NaicsServices = NaicsServices
 
   @Getter getBusinessInformation!: BusinessInformationIF
   @Getter getCurrentNaics!: NaicsIF
   @Getter getEntitySnapshot!: EntitySnapshotIF
   @Getter hasNatureOfBusinessChanged!: boolean
 
   @Action setNaics!: ActionBindingIF
+  @Action setValidComponent!: ActionBindingIF
 
   /** The naics data on record for the business. */
-  private get originalNaics (): NaicsIF {
+  get originalNaics (): NaicsIF {
     return {
       naicsCode: this.getEntitySnapshot.businessInfo.naicsCode,
       naicsDescription: this.getEntitySnapshot.businessInfo.naicsDescription
     }
   }
 
-  private get haveNaics (): boolean {
+  get haveNaics (): boolean {
     return (!!this.getBusinessInformation.naicsCode && !!this.getBusinessInformation.naicsDescription)
   }
+
+  onEditingChanged (event): void {
+    this.setValidComponent({ key: 'isValidNatureOfBusiness', value: event })
+  }
 }
 </script>
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
 ::v-deep {
+  .invalidSection .col-sm-3:first-child label {
+    color: $app-red !important;
+  }
+
+  .v-input .v-label {
+    font-weight: normal;
+  }
+
```

Comment:
Would you mind making these changes to the shared component so we can all benefit from them? (we=me, haha)

Reply:
You bet, will do!

Reply:
Created!

Reply:
https://github.com/bcgov/bcrs-shared-components/pull/119